exercises/chapter10/test/Main.purs

```diff
@@ -231,19 +231,19 @@ runChapterExamples =
         $ square 5.0
     test "diagonal" do
       Assert.equal 5.0
-        $ runFn2 diagonal 3.0 4.0
-    test "diagonalNested" do
-      Assert.equal 5.0
-        $ diagonalNested 3.0 4.0
+        $ diagonal 3.0 4.0
     test "diagonalArrow" do
       Assert.equal 5.0
         $ diagonalArrow 3.0 4.0
+    test "diagonalUncurried" do
+      Assert.equal 5.0
+        $ runFn2 diagonalUncurried 3.0 4.0
     test "uncurriedAdd" do
       Assert.equal 13
-        $ runFn2 uncurriedAdd 3 10
+        $ uncurriedSum
     test "curriedAdd" do
       Assert.equal 13
-        $ curriedAdd 3 10
+        $ curriedSum
```

Comment:
I think it's clearer to just keep the original lines, rather than testing what value was stored in the `*Sum` variables.

Reply:
My intention was to have _coverage_ on all top-level items. This kind of caught both birds with one stone.

The `*Sum` values are not only to be _snippeted_ into the book. The interested reader could also go looking for the generated code.

For the sake of coverage, I'll add extra tests that assert their value.